tasks/unpause-safe.ts

```diff
@@ -1,11 +1,15 @@
 import { task } from "hardhat/config";
+import { getDeployOptions } from "./args/deployOptions";
 
-task("unpause-safe", "Unpause the safe SC").setAction(async (taskArgs, hre) => {
-  const [adminWallet] = await hre.ethers.getSigners();
-  const fs = require("fs");
-  const config = JSON.parse(fs.readFileSync("setup.config.json", "utf8"));
-  const safeAddress = config["erc20Safe"];
-  const safeContractFactory = await hre.ethers.getContractFactory("ERC20Safe");
-  const safe = safeContractFactory.attach(safeAddress).connect(adminWallet);
-  const result = await safe.unpause();
-});
+task("unpause-safe", "Unpause the safe SC")
+  .addOptionalParam("price", "Gas price in gwei for this transaction", undefined)
+  .setAction(async (taskArgs, hre) => {
+    const [adminWallet] = await hre.ethers.getSigners();
+    const fs = require("fs");
+    const config = JSON.parse(fs.readFileSync("setup.config.json", "utf8"));
```

Comment:
Not sure, would it be better to have a function such as "loadSetupConfig()" defined somewhere in a common place?